src/backend/base/langflow/components/nvidia/nemo_evaluator.py

```diff
@@ -191,26 +167,121 @@ class NVIDIANeMoEvaluatorComponent(Component):
         ),
     ]
 
-    def fetch_models(self):
-        """Fetch models from the specified API endpoint and return a list of model names."""
-        model_url = f"{self.model_base_url}/v1/models"
+    # Inputs for Custom Evaluation
+    llm_as_judge = [
+        MultiselectInput(
+            name="evaluation_stages",
+            display_name="Evaluation Stages",
+            info="List of Evaluation Stages for evaluation.",
+            options=["generation", "judgement"],
+            value=["generation", "judgement"]
+        ),
+        # Conditional inputs for run_inference = True
+        IntInput(
+            name="tokens_to_generate",
+            display_name="Tokens to Generate",
+            info="Max number of tokens to generate during inference.",
+            value=1024
+        ),
+        SliderInput(
+            name="temperature",
+            display_name="Temperature",
+            range_spec=RangeSpec(min=0.0, max=1.0, step=0.01),
+            min_label="Precise",
+            max_label="Creative",
+            value=0.1,
+            info="The temperature to be used during generation sampling (0.0 to 2.0)."
+        ),
+        IntInput(
+            name="top_k",
+            display_name="Top_k",
+            info="Top_k value for generation sampling.",
+            value=1
+        ),
+        FloatInput(
+            name="top_p",
+            display_name="Top_p",
+            info="Top_p value for generation sampling.",
+            value=1.0
+        ),
+        # Conditional inputs for run_inference = True
+        DropdownInput(
+            name="judge_llm_name",
+            display_name="LLM Name for judge",
+            info="Select the model for evaluation judge",
+            options=[],  # Dynamically populated
+            refresh_button=True
+        ),
+        # Conditional inputs for run_inference = True
+        IntInput(
+            name="judge_tokens_to_generate",
+            display_name="Judge Tokens to Generate",
+            info="Max number of tokens to generate during inference for the judge.",
+            value=1024
+        ),
+        SliderInput(
+            name="judge_temperature",
+            display_name="Temperature for Judge",
+            range_spec=RangeSpec(min=0.0, max=1.0, step=0.01),
+            min_label="Precise",
+            max_label="Creative",
+            value=0.1,
+            info="The temperature to be used during generation sampling (0.0 to 2.0) for the judge."
+        ),
+        IntInput(
+            name="judge_top_k",
+            display_name="Top_k for Judge",
+            info="Top_k value for generation sampling for the judge.",
+            value=1
+        ),
+        FloatInput(
+            name="judge_top_p",
+            display_name="Top_p for Judge",
+            info="Threshold to select from most probable tokens until cumulative probability exceeds this value",
+            value=1.0
+        ),
+        MessageTextInput(
```

Comment:
looks like import is missing. But also I don't understand why this is here. Are you mapping this into the "input_dir"?